src/components/scrollbar/Scrollbar.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { SScrollbar } from './styles/SScrollbar';
+
+export const Scrollbar: React.FC = ({ ...props }) => {
```

Comment:
@steiningermona sorry for not being clear on this one. What I meant was that the ref is forwarded from the upper component, so you have to implement this like this:

```
const Scrollbar = React.forwardRef((props, ref) => (

)
```

Also you don't need to destructure the props and then use the spread, just use props parameter.

Instead of
({ ...props }) => ...

Just use
(props) => ...